### ISSUE TITLE
Slide Show 2.0.0-beta.3

### ISSUE THREAD

**Issue by ricklach** [fisharebest/webtrees#2489]
The slide show module is not working on my website. There is no response to clicking on either button.

**Comment by fisharebest** [comment]
Perhaps you have no suitable media files to show?

If I visit your site (not logged in), I can only see two media objects.  One is a text file, and one is not linked to any individual.

Maybe your privacy rules are preventing media from being displayed?

**Comment by samiona2** [comment]
I have many media objects, but my slideshow is empty. For now it is limited to logged in users, but the media list shows them all, but the slideshow shows none. I see no privacy rules that would limit this.

**Comment by fisharebest** [comment]
Perhaps you have no media objects that fulfil the criteria needed for the slideshow block.

Media type?
Media format?
Linked to individual?
Linked to fact/event?

**Comment by samiona2** [comment]
@fisharebest 
Here is a sample snippet (I hope that my sanitizing it did not ruin the format). It has the format but not type. The slideshow works just fine in 1.7.8
```
0 @I4@ INDI
1 NAME John /Doe/
2 GIVN John
2 SURN Doe
1 SEX M
1 BIRT
2 DATE 24 Jul 1930
2 PLAC some place
2 SOUR @S815@
2 SOUR @S1218@
2 SOUR @S17@
1 RESI 123 main
2 DATE From 1939 to 1950
2 PLAC some place
2 SOUR @S1383@
3 PAGE 1940 bla bla
3 OBJE
4 FORM jpg
4 FILE blabla.jpg
4 TITL some title
4 _PRIM Y
2 SOUR @S1383@
3 PAGE some page
3 OBJE
4 FORM jpg
4 FILE somepic.jpg
4 TITL some title
4 _PRIM Y
2 SOUR @S1107@
1 CHAN
2 DATE 10 Apr 2013
3 TIME 23:11
1 FAMS @F3@
1 FAMC @F6@
```

**Comment by fisharebest** [comment]
> Media type?

Your media does not have a type.

The slide show only shows media with specified types (e.g. "photo").

**Comment by fisharebest** [comment]
> The slide show only shows media with specified types (e.g. "photo").

Since this is working as designed, I'm going to close this issue.

If you think the design is wrong, perhaps discuss it on the forum or raise another issue.

**Comment by samiona2** [comment]
Is there any reason that the slideshow only shows media type photo? Working as designed is nice, but is there a logical reason not to show scans etc?

**Comment by fisharebest** [comment]
> Is there any reason that the slideshow only shows media type photo?

Is shows any media type that you select - including scans

> The slideshow works just fine in 1.7.8

The 1.7 code always included media without a type.

This was reported as a bug, and fixed in 2.0.  Someone complained that they selected only "photos" and were seeing media that was not a photograph.

We have a type of "other".  When this is selected, we can also including media without a type.
